services/bundle_analysis/new_notify/messages/comment.py

```diff
@@ -50,6 +68,21 @@ def build_message(self, context: BundleAnalysisPRCommentNotificationContext) ->
         )
         return template.render(context)
 
+    @sentry_sdk.trace
+    def build_upgrade_message(
+        self, context: BundleAnalysisPRCommentNotificationContext
+    ) -> str:
+        template = loader.get_template("bundle_analysis_notify/upgrade_comment.md")
+        context = UpgradeCommentTemplateContext(
+            activation_link=get_members_url(context.pull.database_pull),
+            codecov_instance="self_hosted" if requires_license() else "cloud",
+            codecov_name="your instance of Codecov"
+            if requires_license()
+            else "Codecov",
```

Comment:
Both of these depend on `requires_license()`. I would rather define a `is_saas: bool` field for the template and match on that. In particular, the template already matches on `codecov_instance` in a bunch of places, but the `codecov_name` is being resolved in python code instead.